src/constants/text.js

```diff
@@ -30,6 +30,8 @@ export default {
   tabParade: "Parade",
   tabSaved: "Saved",
   tabSupportUs: "Support us",
+  tabParadeMap: "Map",
```

Comment:
Could you add some e2e cases for this new navigation? Just make sure you can tap into all the screens from opening the app. There's an existing test doing this for the tabs we already have

Reply:
I will get on that now :)